<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryViewDlg</class>
 <widget class="QDialog" name="HistoryViewDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Historial de cambios</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/resources.qrc">
    <normaloff>:/aplicacion/images/alephLogoIcon.ico</normaloff>:/aplicacion/images/alephLogoIcon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Historial de Cambios</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableView" name="tvHistoryView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Detalle de la entrada seleccionada</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="txtData"/>
   </item>
   <item>
    <widget class="QPushButton" name="pbOk">
     <property name="text">
      <string>Ok</string>
     </property>
     <property name="icon">
      <iconset resource="../../../resources/resources.qrc">
       <normaloff>:/aplicacion/images/ok.png</normaloff>:/aplicacion/images/ok.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/resources.qrc"/>
  <include location="../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
